### DISCUSSION TITLE
Domain configuration crashed from too much traffic?

### DISCUSSION THREAD

**Discussion by aksperiod** [vercel/vercel#5056] [Help] [answered]
Yesterday my app had a relatively large influx (500-1000) of simultaneous visitors in a couple of seconds. Seconds later, the domain routing crashed and the site could not be accessed. The routing (which I think might have been misconfigured) was set as an A NAME pointing to `35.235.101.253` from AWS's Route 53. The actual application server was still up at the `.vercel.app` domain, so the problem was the routing.

When I looked back at the project's domain seetings, it gave me a new miconfiguration error saying that the domain should point to `76.76.21.21` instead. However, before the crash the site had been working fine using `35.235.101.253`. 

I'm wondering if anyone has any idea of _what broke_ since the app was up and also whether `76.76.21.21` will fix that problem, or whether I should roll my own ec2's with a load balancer instead of relying on vercel's hosting.

**Comment by aksperiod** [comment]
Okay after talking to Vercel support it seems that `35.235.101.253` is an internal IP that was automatically rotated due to the bump in traffic, which is why it was working before then.

**Comment by styfle** [comment, marked as answer]
Hi @aksperiod 

We have never documented internal IP's before because they could change at any time.

We recently announced our new Edge Infrastructure which includes support for `76.76.21.21` so you can rely on this IP.

Read more: https://vercel.com/blog/new-edge-dev-infrastructure
